proxy/src/coco.rs

```diff
@@ -290,6 +290,24 @@ pub fn commit(paths: &Paths, id: &str, sha1: &str) -> Result<Commit, error::Erro
     Ok(Commit::from(commit))
 }
 
+/// Retrieves the [`Commit`] history for the given `branch`.
+///
+/// # Errors
+///
+/// Will return [`error::Error`] if the project doesn't exist or the surf interaction fails.
+pub fn commits(paths: &Paths, id: &str, branch: &str) -> Result<Vec<Commit>, error::Error> {
```

Comment:
Here we rather want a `revision`, similar to how the `tree` function works as it is equally valid to look at a history for a tag or even a commit. That we only uses with branches for now shouldn't affect this API.

Reply:
The coco function takes a branch name though, so not sure that makes sense.

Reply:
This should change in coco then.

Reply:
Yeah, that's probably right.